inc/common/purge.php

```diff
@@ -436,7 +436,21 @@ function do_admin_post_rocket_purge_opcache() {
 		return;
 	}
 
-	rocket_reset_opcache();
+	$reset_opcache = rocket_reset_opcache();
+
+	if ( ! $reset_opcache ) {
+		$op_purge_result = array(
+			'result'  => 'error',
+			'message' => __( 'OPcache purge failed.', 'rocket' ),
+		);
+	} else {
+		$op_purge_result = array(
+			'result'  => 'success',
+			'message' => __( 'OPcache successfully purged', 'rocket' ),
+		);
+	}
+
+	set_transient( $GLOBALS['current_user']->ID . '_opcache_purge_result', $op_purge_result );
```

Comment:
```suggestion
	set_transient( get_current_user_id() . '_opcache_purge_result', $op_purge_result );
```

Reply:
Replaced all `$GLOBALS['current_user']->ID` with `get_current_user_id()` in the files:
`/inc/common/purge.php`
`/inc/admin/ui/notices.php`